packages/aws-cdk/lib/legacy-exports-source.ts

```diff
@@ -0,0 +1,29 @@
+export * from './legacy-logging-source';
```

Comment:
Needs an explanation on why this file exists and the export goes through `index.ts` (because of bundle size).